h5py_wrapper/wrapper.py

```diff
@@ -282,9 +284,12 @@ def _evaluate_key(f):
     Evaluate the key of f and handle non-string data types.
     """
     name = os.path.basename(f.name)  # to return only name of this level
-    if ('_key_type' in f.attrs and
-            f.attrs['_key_type'] not in ['str', 'unicode', 'string_']):
-        name = ast.literal_eval(name)
+    if '_key_type' in f.attrs:
```

Comment:
should we use the same logic, i.e., `try ... except` here, as we do above with `value_type`?